src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -70,16 +70,63 @@
                     <ComboBox x:Name="DefaultTerminal"
-                              x:Load="False"
-                              ItemsSource="{x:Bind DefaultTerminals, Mode=OneWay}"
-                              SelectedItem="{x:Bind CurrentDefaultTerminal, Mode=TwoWay}"
-                              Style="{StaticResource ComboBoxSettingStyle}" />
+                              ItemsSource="{x:Bind State.Settings.DefaultTerminals, Mode=OneWay}"
+                              SelectedItem="{x:Bind State.Settings.CurrentDefaultTerminal, Mode=TwoWay}"
+                              Style="{StaticResource ComboBoxSettingStyle}">
+                        <ComboBox.ItemTemplate>
+                            <DataTemplate x:DataType="SettingsModel:DefaultTerminal">
+                                <Grid HorizontalAlignment="Stretch"
+                                      VerticalAlignment="Stretch"
+                                      ColumnSpacing="8">
+
+                                    <Grid.ColumnDefinitions>
+                                        <!--  icon  -->
+                                        <ColumnDefinition Width="24" />
+                                        <!--  profile name and author  -->
+                                        <ColumnDefinition Width="*" />
+                                        <!--  version  -->
+                                        <ColumnDefinition Width="48" />
+                                    </Grid.ColumnDefinitions>
+
+                                    <Grid.RowDefinitions>
+                                        <!--  profile name  -->
+                                        <RowDefinition Height="*" />
+                                        <!--  author and version  -->
+                                        <RowDefinition Height="*" />
+                                    </Grid.RowDefinitions>
+
+                                    <IconSourceElement Grid.Row="0"
+                                                       Grid.RowSpan="2"
+                                                       Grid.Column="0"
+                                                       Width="24"
+                                                       Height="24"
+                                                       VerticalAlignment="Center"
+                                                       IconSource="{x:Bind Icon, Mode=OneWay, Converter={StaticResource IconSourceConverter}}" />
+
+                                    <TextBlock Grid.Row="0"
+                                               Grid.Column="1"
+                                               Grid.ColumnSpan="2"
+                                               Text="{x:Bind Name}" />
+
+                                    <TextBlock Grid.Row="1"
+                                               Grid.Column="1"
+                                               Foreground="{StaticResource SystemBaseMediumColor}"

```

Comment:
Does this work in both dark and light mode? What if the OS is set to dark mode, but the app is set to light?

Reply:
I checked OS set to light and OS set to dark with the app following. If you switch it while it is open, it looks terrible. But if you close and reopen the page it's fine. I wasn't going to cry over that.

I can check and cross-reference the other bug I guess.